### ISSUE TITLE
a11y_Tag> Disabled Inverse> 1 violations is appearing

### ISSUE THREAD

**Issue by priya-rajput** [cengage/react-magma#521]
**Describe the bug**
Tag>> "1 violations" is appearing for "Disabled Inverse". 

**To Reproduce**
1. Go to 'https://storybook-preview-290--upbeat-sinoussi-f675aa.netlify.app/?path=/story/tag--disabled-inverse'
2. Click on 'Accessibility" add on
3. "1 violations" is appearing

**Expected behavior**
"0 violations" should appear for Tag> Disabled Inverse.

**Screenshots**
https://somup.com/c3n263Tcng

**Desktop :**
 - OS: [Win 10]
 - Browser [chrome]
 - Version [Version 98.0.4758.82 (Official Build) (64-bit)]

**Comment by silvalaura** [comment]
There is no minimum contrast required for disabled, so closing as won't fix.
